### ISSUE TITLE
We need a Code of Conduct

### ISSUE THREAD

**Issue by code4cake** [Kartikkh/WeDonateMobileApp#11]
We need a CoC for those contributing. Here is an example from [facebook](https://code.facebook.com/pages/876921332402685/open-source-code-of-conduct).

**Comment by akshayvinchurkar1** [comment]
You mean Contribution rules? how to contribute, what is current coding style, naming conventions, design standards etc right

**Comment by code4cake** [comment]
@akshayvinchurkar1, yeah pretty much. Just global rules on how anyone contributing should behaved, what to say/not to say, etc. 

Here is another example, [ecma/tc39](https://tc39.github.io/code-of-conduct/) COC

**Comment by davisokoth** [comment]
And is there a discussion board somewhere in case someone has trouble setting up for example? Some not development related probably?

**Comment by Kartikkh** [comment]
I think if there is problem in setting up than , he can raise a issue for that .

**Comment by code4cake** [comment]
@davisokoth just create an issue and label it `discussion`?
